app/src/main/java/com/example/calculator/view/calculator/CalculatorFragmentExtension.kt

```diff
@@ -0,0 +1,62 @@
+package com.example.calculator.view.calculator
+
+import com.example.calculator.R
+import kotlinx.android.synthetic.main.fragment_calculator2.*
+
+fun CalculatorFragment.numberOnclick() {
+    button0.setOnClickListener { numberOnClick(getString(R.string.zero)) }
+    button1.setOnClickListener { numberOnClick(getString(R.string.one)) }
+    button2.setOnClickListener { numberOnClick(getString(R.string.two)) }
+    button3.setOnClickListener { numberOnClick(getString(R.string.three)) }
+    button4.setOnClickListener { numberOnClick(getString(R.string.four)) }
+    button5.setOnClickListener { numberOnClick(getString(R.string.five)) }
+    button6.setOnClickListener { numberOnClick(getString(R.string.six)) }
+    button7.setOnClickListener { numberOnClick(getString(R.string.seven)) }
+    button8.setOnClickListener { numberOnClick(getString(R.string.eight)) }
+    button9.setOnClickListener { numberOnClick(getString(R.string.nine)) }
+}
+
+fun CalculatorFragment.operatorListener() {
+    buttonClear.setOnClickListener { clear() }
+    buttonPlus.setOnClickListener { operatorOnClick(getString(R.string.plus)) }
+    buttonMinus.setOnClickListener { operatorOnClick(getString(R.string.minus)) }
+    buttonMultiply.setOnClickListener { operatorOnClick(getString(R.string.multiply)) }
```

Comment:
set click nhiều thế này thì anh nên implement View.OnClickListener rồi setOnClickListener(this)